src/Homotopy/Truncation/SetTruncation.ard

```diff
@@ -0,0 +1,14 @@
+\import Homotopy.Truncation
+
+\data SetTrunc (A : \Type)
+  | point A
+  | trunc (x y : SetTrunc A) (p q : x = y) (i j : I) \elim i, j {
+    | left, j => p @ j
+    | right, j => q @ j
+    | i, left => x
+    | i, right => y
```

Comment:
Two additional definitional equalities that aren't written in Agda but seem to exist.